.github/workflows/ci-main.win.yml

```diff
@@ -25,6 +25,10 @@ jobs:
       - name: Install dependencies
         run: yarn install
 
+      # Delete NOTICE.txt because it's too big for yarn to handle.
+      - name: Delete NOTICE.txt
```

Comment:
Consider expanding the comment to reference the open yarn bug causing this behavior: https://github.com/yarnpkg/yarn/issues/7783